File: src/chrome/android/java/src_wooeen_base/com/wooeen/model/to/CategoryTO.java
package com.wooeen.model.to;

public class CategoryTO {
    private int id;
    private String name;
    private String description;
    private int totalAdvertisers;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getTotalAdvertisers() {
        return totalAdvertisers;
    }

    public void setTotalAdvertisers(int totalAdvertisers) {
        this.totalAdvertisers = totalAdvertisers;
    }
}

File: src/brave/ios/brave-ios/Sources/Wooeen/Model/CategoryTO.swift
import Foundation

public struct CategoryTO: Decodable {
    public var id: Int
    public var name: String?
    public var description: String?
    public var totalAdvertisers: Int?
}

public struct CategoryResponse: Decodable{
    public var result: Bool
    public var callback: [CategoryTO]?
    public var message: String?
}
